bundles/af-core/src/main/java/com/adobe/cq/forms/core/components/models/form/FormContainer.java

```diff
@@ -362,4 +362,24 @@ default void visit(Consumer<ComponentExporter> callback) throws Exception {}
     default String getParentPagePath() {
         return null;
     }
+
+    @JsonProperty("enableAutoSave")
+    default boolean isEnableAutoSave() {
```

Comment:
This would also get fixed via AutoSaveConfig, why do you have json property here, when its part of json ignore

```
  @JsonIgnore
    public AutoSaveConfig getAutoSaveConfig() {
        return autoSaveConfig;
    }

```

Reply:
Done